app/objective/post.controller.js

```diff
@@ -20,17 +44,23 @@ const validationRules = () => {
       .trim()
       .escape(),
     body('needs')
+      .custom(noNeeds)
+      .withMessage(BLANK_ERROR)
+      .bail()
       .isLength({ min: 1 })
       .withMessage('Select the needs for this objective'),
     body('needs').toArray(),
+    body('noNeedsConfirmation')
```

Comment:
You can stick the default message in the body call:
`body('noNeedsConfirmation', 'You must confirm that you have completed the OASys risk assessment')`